### PR TITLE
INTLY-4201

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**JIRA**
https://issues.redhat.com/browse/INTLY-4201

ping @mikenairn

### CODE REVIEW

File: ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+promotion:
+  namespace: integr8ly
+  name: heimdall
+
+tag_specification:
+  name: '4.2'
+  namespace: ocp
+
+tests:
```
Comment (laurafitzgerald) [inline ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml]: Not sure if these are all the tests we want. I've looked at the other integreatly projects to see what they're doing there. There may be others we want here.

File: ci-operator/jobs/integr8ly/heimdall/integr8ly-heimdall-master-postsubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+postsubmits:
```
Comment (laurafitzgerald) [inline ci-operator/jobs/integr8ly/heimdall/integr8ly-heimdall-master-postsubmits.yaml]: This was generated by using the prow generator steps described [here](https://github.com/openshift/release/tree/master/ci-operator#ci-operator) I'm not sure if these are right or how I'd go about testing it.

File: ci-operator/jobs/integr8ly/heimdall/integr8ly-heimdall-master-presubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,194 @@
+presubmits:
```
Comment (laurafitzgerald) [inline ci-operator/jobs/integr8ly/heimdall/integr8ly-heimdall-master-presubmits.yaml]: Same here

File: ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml (hunk 1 above)
Comment (mikenairn) [reply ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml]: We want format,unit and e2e for now https://github.com/openshift/release/blob/master/ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml#L48-L59

You need make sure all the targets exist in the Makefie first though https://github.com/integr8ly/heimdall/blob/master/Makefile

File: ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/integr8ly/heimdall/build
+  dockerfile_path: build/Dockerfile
+  to: heimdall
+
+build_root:
+  image_stream_tag:
+    name: heimdall-base-image
```
Comment (mikenairn) [inline ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml]: We will need to add this

File: ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    namespace: integr8ly
+    tag: latest
+
+binary_build_commands: make code/compile COMPILE_TARGET="./build/_output/bin/heimdall"
```
Comment (mikenairn) [inline ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml]: This make target needs to be added

File: ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml (hunk 4 above)
Comment (laurafitzgerald) [reply ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml]: 👍 Where should I add it?

File: ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml (hunk 5 above)
Comment (laurafitzgerald) [reply ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml]: Done

File: ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml (hunk 1 above)
Comment (laurafitzgerald) [reply ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml]: format unit and e2e added. Resolving.

**Pull request merged**